packages/CollabCodeEdit-Core.package/CCEMessage.class/class/forwardEditMessage..st

```diff
@@ -1,4 +1,4 @@
 server-Message creation
-forwardEditMessage: aSerializeddCCEEdit
+forwardEditMessage: aCCEEdit
```

Comment:
```suggestion
forwardEdit: aCCEEdit
```
to stay within the naming convention of CCEMessage. Because every class call lookes like
`CCEMessage ackJoinSession:` etc. it looks a bit odd to append an extra `Message` to every class method.

Reply:
When this method was named there was an assumption that the Server wouldn't be able to understand the edit, and never actually touch the edit. Within the old assumption the name is valid, since it implies the servers lack of CCEEdit interpretation, just forwarding the message instead.

That assumption has since changed, so a renaming would be in order.